content/resources/user/submit-data/sower.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Gen3 - Data Contribution"
+date: 2018-09-12T20:53:20-05:00
+linktitle: /resources/user/submit-data
+layout: withtoc
+menuname: userMenu
+---
+{{% markdownwrapper %}}
+# Linking Data from external Data Clouds to Gen3 Data Commons
+***
+
+It is possible to link data on Gen3 that is stored on other cloud services (Amazon Web Services AWS, Google Cloud Storage GCS) by a process called DIIRM (*Data Ingestion, Integration, and Release Management*). In other words, if you have a bucket of files and want to link the data to Gen3 you can find below a step-by-step guide and a practical example. Before going forward, you need to make sure you have access to the external bucket.
```

Comment:
`Data files that are stored on other cloud services (Amazon Web Services AWS, Google Cloud Storage GCS) can be made available in a Gen3 data commons without moving it. Below is a step-by-step guide with a practical example. 

Note: Before going forward, you need to make sure you have access to the external bucket.`

Reply:
done